fish_prompt.fish

```diff
@@ -374,12 +374,12 @@ set -g CMD_DURATION 0
   #end
 
   #current_gemset alternativ
-  #  else if test (rbenv gemset active >/dev/null ^&1) = "no active gemsets" # not sure what ^&1
+  #  else if test (rbenv gemset active >/dev/null 2>&1) = "no active gemsets" # not sure what 2>&1
```

Comment:
I guess there could be a previous commit removing this dead code.

Reply:
it's @eugenesvk's repo, and @eugenesvk's dead code. i wouldn't presume to remove it, because i don't know why it's still there :)

Reply:
Fair. Agreed.